backend/corpora/lambdas/api/v1/collection_uuid/upload.py

```diff
@@ -24,6 +24,8 @@ def link(collection_uuid: str, body: dict, user: str):
         resp = dropbox.get_file_info(url)
     except requests.HTTPError:
         raise InvalidParametersHTTPException("The URL provided causes an error with Dropbox.")
+    except KeyError:
```

Comment:
Can you add a test case for this?

Reply:
This was changed, and a test was added to verify the new error condition